app/models/miq_provision_virt_workflow.rb

```diff
@@ -339,8 +339,8 @@ def allowed_templates(options = {})
       end
     end
 
-    # Only select the colums we need
-    vms = vms.select(:id, :type, :name, :guid, :uid_ems, :ems_id, :cloud_tenant_id)
+    # Only select the colums we need on non-deprecated templates
+    vms = vms.where(:deprecated => false).select(:id, :type, :name, :guid, :uid_ems, :ems_id, :cloud_tenant_id)
```

Comment:
@agrare Is deprecated used by any other provider?

Reply:
@jrafanie Will this handle nil columns as well?

Reply:
yeah, I can check for `where.not(:deprecated => true)` or something to get only falsey values

Reply:
Awesome, the existing tests failed in this exact area.  Yay tests!

Reply:
IBM Power was going to start using it but currently it is only set by google cloud